src/main/java/com/snoop/controllers/WorkspaceController.java

```diff
@@ -57,7 +57,7 @@ public String index(Map<String, Object> model) {
 
 		logger.debug("index() is executed!");
 
-		Workspace wkps = new Workspace(new Long(1));
+		Workspace wkps = new Workspace(Long.valueOf(1));
 		
 		workspaceRepository.save(wkps);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this call to "equals"; comparisons between unrelated types always return false. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2159)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00108)